bundles/org.openhab.binding.zm/src/main/java/org/openhab/binding/zm/internal/handler/ZmMonitorHandler.java

```diff
@@ -0,0 +1,293 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.zm.internal.handler;
+
+import static org.openhab.binding.zm.internal.ZmBindingConstants.*;
+
+import java.time.ZonedDateTime;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import javax.measure.quantity.Time;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.i18n.TimeZoneProvider;
+import org.eclipse.smarthome.core.library.types.DateTimeType;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.library.types.RawType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.library.unit.SmartHomeUnits;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandlerService;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.zm.action.ZmActions;
+import org.openhab.binding.zm.internal.config.ZmMonitorConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link ZmMonitorHandler} represents a Zoneminder monitor. The monitor handler
+ * interacts with the server bridge to communicate with the Zoneminder server.
+ *
+ * @author Mark Hilbush - Initial contribution
+ */
+@NonNullByDefault
+public class ZmMonitorHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(ZmMonitorHandler.class);
+
+    private final TimeZoneProvider timeZoneProvider;
+
+    private @Nullable ZmBridgeHandler bridgeHandler;
+
+    private @NonNullByDefault({}) String monitorId;
+    private @Nullable Integer imageRefreshInterval;
+    private Integer alarmDuration = DEFAULT_ALARM_DURATION_SECONDS;
```

Comment:
It's good practice to append the unit to the field name e.g. alarmDurationSec.

Reply:
Done